dependencies.yaml

```diff
@@ -99,27 +132,29 @@ dependencies:
               - python>=3.9,<3.11
   run_python:
     common:
-      - output_types: [conda, requirements]
+      - output_types: [conda, requirements, pyproject]
         packages:
           - numba>=0.57
           - numpy>=1.21
           - pandas>=1.3,<1.6.0dev0
           - pynvml>=11.0.0,<11.5
-          - rapids-dask-dependency=24.02.*
+          - rapids-dask-dependency==24.02.*
           - zict>=2.0.0
   test_python:
     common:
-      - output_types: [conda]
+      - output_types: [conda, requirements, pyproject]
         packages:
-          - cudf=24.02
-          - dask-cudf=24.02
-          - kvikio=24.02
+          - cudf==24.02.*
+          - dask-cudf==24.02.*
+          - kvikio==24.02.*
           - pytest
           - pytest-cov
+          - ucx-py==0.36.*
+      - output_types: [conda]
+        packages:
+          - distributed-ucxx==0.35.*
           - ucx-proc=*=gpu
-          - ucx-py=0.36
-          - ucxx=0.35
-          - distributed-ucxx=0.35
+          - ucxx==0.35.*
```

Comment:
Should be `0.36`

Reply:
Good catch. Looks like these were previously missing in update-version.sh. I've added them and also fixed all the version specifiers there to use `==` which is PEP 508 compliant.